Find a functionally equivalent code:com.android.internal.util.XmlUtils.readMapXml:COMMENT
Method Modifier: public      static      final       internal    
Comment:/**
 * Read a HashMap from an InputStream containing XML.  The stream can
 * previously have been written by writeMapXml().
 *
 * @param in The InputStream from which to read.
 *
 * @return HashMap The resulting map.
 *
 * @see #readListXml
 * @see #readValueXml
 * @see #readThisMapXml
 * #see #writeMapXml
 */

@@ -1,5 +1,5 @@
 {
     XmlPullParser parser = Xml.newPullParser();
     parser.setInput(in, null);
-    return (HashMap) readValueXml(parser, new String[1]);
+    return (HashMap<String, ?>) readValueXml(parser, new String[1]);
 }

